### PR TITLE
prow: Drop some plugins from coreos/rpm-ostree

### PR DESCRIPTION
**Pull request by cgwalters**
Most notably drop approve and lgtm, which are responsible
for a lot of comment spam.  We are going to experiment
with using the Github-native methods for this.

xref https://github.com/coreos/fedora-coreos-tracker/issues/764

### CODE REVIEW

**Comment by cgwalters** [comment]
/hold
for approval from another coreos/rpm-ostree developer by tomorrow.

**Comment by cgwalters** [comment]
To be explicit, after this merges PRs to the rpm-ostree repo would use the Github Review Changes -> "Approve" flow for the equivalent of "/approve" and the "Merge after tests pass" button (which...hmm, I think we need a repo admin to enable).

**Comment by jlebon** [comment]
Definitely interested to try this out!
/lgtm

**Review by kelvinfan001 — APPROVED** [review]
I think this is worth experimenting with. I always thought the Github flow was more intuitive and user-firendly, and the reduction in spam feels a lot cleaner too :)

**Review by alvaroaleman — APPROVED** [review]

**Review by travier — APPROVED** [review]
Let's give it a try!

**Pull request merged**
